common/app/model/Section.scala

```diff
@@ -63,3 +63,31 @@ object SectionId {
 
   def fromId(id: String): SectionId = SectionId(id)
 }
+
+object Neilsen {
```

Comment:
There are more Guardian sections than the ones listed here. I'm still waiting for feedback from Nielsen on how the other sections should be mapped to APIDs.